SparkPost/2web2ui
# FE-1109 - Incorporate UX feedback for the Suppressions feature

[FE-1109](https://jira.int.messagesystems.com/browse/FE-1109)

### What Changed

- Joined the Suppressions filtering and the table collection
- Added `minWidth` to Hibana date picker `<Select/>` only
- Remove used of `<LongTextContainer/>` within Suppressions details modal

### How To Test

- Go to `/list/suppressions` and verify the above
- Check _both_ the OG and Hibana themes

### To Do
